client/components/plans/plan/index.jsx

```diff
@@ -83,6 +88,11 @@ module.exports = React.createClass( {
 				'is-plan-specific': feature.planSpecific
 			} );
 
+			if ( abtest( 'plansFeatureList' ) === 'andMore' && feature.testVariable ) {
```

Comment:
`testVariable` is still needed here to keep the feature from being shown in the main list, but you're right that we could check for the Business plan to set `showMoreLink`.